MAKEALL

```diff
@@ -4,7 +4,7 @@ BASE_FOLDER="${BASE_FOLDER:-/tmp/upstream-boot-build/}"
 BUILD_FOLDER=${BASE_FOLDER}/build
 DEPLOY_FOLDER=${BASE_FOLDER}/deploy
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
-LSDEFCONFIGS=`cd $SCRIPT_DIR/configs;ls|sort`
+LSDEFCONFIGS=`cd $SCRIPT_DIR/configs;ls *_defconfig|sort`
```

Comment:
You could also move the uenv.txt files to a new folder called `uenv` or something, leave the configs dir for just defconfigs.

Reply:
Yes, I considered that.  But I thought it better to restrict the config files to those matching our filename pattern (*_defconfig).  Plus, I didn't want to create a new directory for just 1 file and I don't see any further uses.

Reply:
That's fine, and we can always add the `uenvs` directory if we get any more `uenv.txt` files as that point.